spec/models/project_spec.rb

```diff
@@ -293,17 +293,39 @@
   end
 
   describe '.goal_between' do
-    let(:start_at) { 100 }
-    let(:ends_at) { 200 }
-    subject { Project.goal_between(start_at, ends_at).order(:id) }
+    let(:start_at)                 { 200 }
+    let(:ends_at)                  { 400 }
+    let!(:all_projects)            { projects_under_start_at + projects_over_ends_at + projects_goal_between }
```

Comment:
As @Maychell said, the order doesn't matter, when one of those "variables" defined by a `let` is called, all of the others are already defined, only not instantiated.

The defined `let`' s behave similar to something like that:
```
def all_projects
  @all_projects ||= projects_under_start_at + ...
end

def projects_under_start_at
  @projects_under_start_at ||= create_list(:project, 3, goal: 199)
end

...
```